components/kubearchive/base/kustomization.yaml

```diff
@@ -132,6 +132,12 @@ patches:
                 runAsNonRoot: true
               ports:
                 - containerPort: 8081
+              resources:
+                resources:
```

Comment:
shouldn't this second `resources` be `requests`

```suggestion
                requests:
```

Reply:
You are totally right, thanks!

Reply:
Weirdly, kubelinter does not flag it as an error.